metricflow/model/model_transformer.py

```diff
@@ -16,24 +19,35 @@ class ModelTransformer:
     Generally used to make it more convenient for the user to develop their model.
     """
 
+    DEFAULT_PRE_VALIDATION_RULES: List[ModelTransformRule] = [LowerCaseNamesRule()]
+
+    DEFAULT_POST_VALIDATION_RULES: List[ModelTransformRule] = [
+        CreateProxyMeasureRule(),
+        BooleanMeasureAggregationRule(),
+        CompositeIdentifierExpressionRule(),
+    ]
```

Comment:
Let's make these tuples so they can't be mutated in the runtime.

Reply:
done